llms/gguf_llm/models.py

```diff
@@ -265,13 +277,17 @@ def load(gguf_file: str, repo: str = None):
     if gguf_ft == 0 or gguf_ft == 1:
         # ALL_F32 or MOSTLY_F16
         quantization = None
+        print("Non-quantized Float32 model")
```

Comment:
This print is not right. It should be either float32 or float16.

Reply:
@awni I have deleted that print, since last condition already prints warning for unsupported quants